docker-compose.override.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.7"
+
+services:
+  api:
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./templates:/app/templates
+      - ./static:/app/static
+      - ./strings.yaml:/app/strings.yaml
+      - ./requirements.txt:/app/requirements.txt
+      - ./generate_sites.py:/app/generate_sites.py
+      - ./download_data.py:/app/download_data.py
```

Comment:
This makes sure we are mounting directories and files from the host into the container. We do this so that we can build the docker image once, and then `make sh` just mounts the host's files into the container.

We can then use host tools, editors, ides, etc. and work in the container without having to re-build the image.

Rebuilding the image is only needed when we e.g. change the requirements.txt.